flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -1061,9 +1064,11 @@ private StateBackend loadStateBackend(ReadableConfig configuration, ClassLoader
 	}
 
 	// private helper for passing different names
-	private <OUT> DataStreamSource<OUT> fromParallelCollection(SplittableIterator<OUT> iterator, TypeInformation<OUT>
-			typeInfo, String operatorName) {
-		return addSource(new FromSplittableIteratorFunction<>(iterator), operatorName, typeInfo);
+	private <OUT> DataStreamSource<OUT> fromParallelCollection(
+			SplittableIterator<OUT> iterator, TypeInformation<OUT>
```

Comment:
nit, weird formatting: the `TypeInformation...` parameter should be on a newline